prompt-note/task/index.js

```diff
@@ -0,0 +1,79 @@
+const prompt = require("prompt-sync")();
+
+let count = 0;
+let listItem = [];
+const resultMessage = () => console.log("종료 되었습니다.");
+
+// 질문
+const question = () => {
+  const message = prompt(
+    "1. 전체메모 조회 / 2. 메모 입력 / 3. 메모 모두 삭제하기 / 4. 메모 수정하기 / 0. 종료 : "
+  );
+  const feedback = Number(message);
```

Comment:
```suggestion
  const feedback = +message;
```
위처럼 + 사용하여 문자 -> 숫자 변경 가능합니당